modules/sso.php

```diff
@@ -623,23 +628,11 @@ function handle_login() {
 		) );
 		$xml->query( 'jetpack.sso.validateResult', $wpcom_nonce, $wpcom_user_id );
 
-		if ( $xml->isError() ) {
-			$error_message = sanitize_text_field(
-				sprintf( '%s: %s', $xml->getErrorCode(), $xml->getErrorMessage() )
-			);
-			JetpackTracking::record_user_event( 'sso_login_failed', array(
-				'error_message' => $error_message
-			) );
-			wp_die( $error_message );
-		}
-
-		$user_data = $xml->getResponse();
-
+		$user_data = $xml->isError() ? false : $xml->getResponse();
 		if ( empty( $user_data ) ) {
-			JetpackTracking::record_user_event( 'sso_login_failed', array(
```

Comment:
In this case, we are removing it completely. I'm not sure that the tracking is that useful in this case since it's a generic error. I can add it back in though :)